nav2_rviz_plugins/include/nav2_rviz_plugins/costmap_cost_tool.hpp

```diff
@@ -48,8 +49,8 @@ class CostmapCostTool : public rviz_common::Tool
 private Q_SLOTS:
 
 private:
-  rclcpp::Client<nav2_msgs::srv::GetCosts>::SharedPtr local_cost_client_;
-  rclcpp::Client<nav2_msgs::srv::GetCosts>::SharedPtr global_cost_client_;
+  std::shared_ptr<nav2_util::ServiceClient<nav2_msgs::srv::GetCosts>> local_cost_client_;
```

Comment:
Use `SharedPtr` you added? Make sure to look around the diffs here in github and change all to use those! :-)